tests/openshift/openshift-tests.yml

```diff
@@ -1,9 +1,37 @@
 ---
 - hosts: localhost
   tasks:
+  
   - name: Test cluster health reports OK
     uri:
-      url: https://ocp.customer2.cor00005.cna.ukcloud.com:8443/healthz/ping
+      url: "{{server}}/healthz/ping"
       status_code: 200
       body: ok
       validate_certs: no
+  
+  - name: Request URL to confirm homepage works
+    uri:
+      url: "{{server}}"
+      status_code: 200
+      validate_certs: no
+
+  - name: Login to openshift
```

Comment:
This is good - loads better with the names, it'll make the playbook output much more useful.